tests/src/test/scala/common/WskTestHelpers.scala

```diff
@@ -31,6 +31,55 @@ import spray.json._
 import java.time.Instant
 
 /**
+ * An arbitrary response of a whisk action. Includes the result as a JsObject as the
+ * structure of "result" is not defined.
+ */
+case class ActivationResponse(result: Option[JsObject], status: String, success: Boolean)
```

Comment:
When a trigger is fired, there will be an activation linking to the rule, whose response does not have result available:
{
    "namespace": "whisk.system",
    "name": "testRule",
    "version": "0.0.1",
    "subject": "whisk.system",
    "activationId": "c2f410c141b9422ca32ab63a7c43d08e",
    "cause": "a8adb44ac0e24b8a9f0486617300b5ae",
    "start": 1502940356583,
    "end": 0,
    "duration": 0,
    "response": {
        "status": "success",
        "statusCode": 0,
        "success": true
    },
    "logs": [],
    "annotations": [],
    "publish": false
}